src/main/scala/cromwell/engine/workflow/WorkflowOptions.scala

```diff
@@ -0,0 +1,89 @@
+package cromwell.engine.workflow
+
+import com.typesafe.config.ConfigFactory
+import cromwell.util.{EncryptedBytes, SecretKey, Aes256Cbc}
+import scala.collection.JavaConverters._
+import spray.json._
+import scala.util.{Try, Success, Failure}
+
+object WorkflowOptions {
```

Comment:
And just to followup as to where I'm going with this ...

The thing which bothers me are all the machinations to sometimes handle the string and sometimes the json object. 

The way I was picturing this was that these special keys would be in a separate table, in which case if we did need to hold mulitple bits of info the table structure could have that. And then WorkflowOptions could be turned into a sum type where each kind knows how to properly extract values